spec/rspec/support/spec/stderr_splitter_spec.rb

```diff
@@ -86,7 +86,7 @@
     splitter.verify_no_warnings!
   end
 
-  unless defined?(RUBY_ENGINE) && RUBY_ENGINE == 'rbx'
+  unless Ruby.rbx?
```

Comment:
Indeed, I didn't mean "please remove this". If you need this on TruffleRuby CI to make sure `rspec-*` jobs pass - please feel free to keep this.

Reply:
Indeed, it's not because truffleruby is not currently tested in rspec-support's CI that it's not useful. It might be tested in some other CI and locally.